### ISSUE TITLE
Weaponcore breaks witha particular set of mods

### ISSUE THREAD

**Issue by LordClownShoes** [sstixrud/WeaponCore#194]
## Is this a multiplayer or singleplayer bug?
SP, have one mod for MP

## Describe the bug:
It appears that a quartet of mods, when added to the group with weaponcore, causes the framework to lose functionality, the mod without these other four present works(such as with the vanilla weapons replacement.) But when dded in the same list, middle mouse click fails to open the weaponcore menu, I've tested these in separate worlds with all the mods I do use, excluding the ones allegedly causing the issue, and in ones where it is just weaponcore and Vanilla weapons replacement, in each instance where the four mods in question are present, it fails to work. I don't think this is an error on weaponcore's part but the mods themselves perhaps not being up to date with the current weaponcore release.

The mods in question:
Mexpex MWI weapon collection 1&2
Mexpex Homing Weapons
Battlecannon and turrets weaponcore MP - This one I
 tested just in case and it appeared to break functionality as well, but not fully confirmed

## Steps To Reproduce:
Simply have these mods loaded in the same world with weaponcore and attempt to use weaponcore functions

## Expected behavior:
I believe these mods normally would work fine, in all truth we could just be waiting for the mod authors to update their mods possibly

### User Error Report (if Weapon-Specific) and log files
-No errors found with the log I generated from MWI.

Zip up that folder and link it here:

**Comment by AtomicHawk** [comment]
My game keeps failing to load completely, and in every SE log file those 4 mods keep coming up with a bunch of errors. I've got a ton of mods loaded so there might be a few issues, but the last time I loaded this save it all worked fine, much less failing to load and kicking errors out. Figured I'd tag on to this thread instead of starting a new one. Log included for reference, although I could be a dunce and the issue could be something else entirely.... 
[SpaceEngineers_20200730_202424886.log](https://github.com/sstixrud/WeaponCore/files/5004401/SpaceEngineers_20200730_202424886.log)
